packages/manager/apps/pci-kubernetes/src/types/index.ts

```diff
@@ -59,16 +82,25 @@ export enum BreakPoints {
   XL = 1140,
 }
 
+// Fonction pour vérifier si une chaîne est une chaîne Base64 valide
+function isBase64(str: string) {
+  try {
+    return btoa(atob(str)) === str;
+  } catch (err) {
+    return false;
+  }
+}
+
```

Comment:
There are multiple issues here:

- we shouldn't declare any logical function into a `types` folder
- the comment is in French
- there is no test associated